recipes/LJSpeech/S2ST/Quantization/utils.py

```diff
@@ -0,0 +1,59 @@
+import itertools
```

Comment:
I feel like this would be better in the train.py unless it is used elsewhere in the recipe

Reply:
Yes, I removed the file